pcsx2/CMakeLists.txt

```diff
@@ -495,8 +495,8 @@ add_custom_command(
 
 # PAD/Linux sources 
 set(pcsx2PADSources
-    PAD/Linux/controller.cpp
-    PAD/Linux/GamePad.cpp
+    PAD/Linux/Device.cpp
+	PAD/Linux/InputManager.cpp
```

Comment:
Formatting, other locations also.

Reply:
Should be taken care of, though the auto-formatting I ran may have tweaked the formatting in a few spots.Still might need to fix up nopch compiling. We'll see.